src/Xamarin.Android.Build.Tasks/Tasks/JdkInfo.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Xamarin.Android.Tools;
+
+namespace Xamarin.Android.Tasks
+{
+	public class JdkInfo : Task
+	{
+		[Output]
+		public string JdkJvmPath { get; set; }
```

Comment:
The addition of this task seems odd; I think it would make more sense to export this property in the [`<ResolveSdks/>` task](https://github.com/xamarin/xamarin-android/blob/f6c3288dd5a7f82c178ef30ad25c11d475f1daf4/src/Xamarin.Android.Build.Tasks/Tasks/ResolveSdksTask.cs), a'la:

```csharp
// Existing; line 84
JavaSdkPath    = MonoAndroidHelper.AndroidSdk.JavaSdkPath;

// New
try {
}
    var info = new JdkInfo (JavaSdkPath);
    JdkJvmPath = info.JdkJvmPath;
catch (Exception e) {
    Log.LogCodedError (...);
}
```

This way we ensure that we're consistent.  By using `JdkInfo.GetKnownSystemJdkInfos()`, it's possible that `<ResolveSdks/>` will provide one JDK path, but `JdkJvmPath` could come from a *different* path.  (Likely? I hope not, but *possible*, which is scary.)

Reply:
Makes sense. That or use `JavaSdkPath` as an input for the task. I will try to extend the ResolveSdks first to see if it doesn't screw up the dependencies.